target/linux/ath79/image/generic.mk

```diff
@@ -84,6 +84,30 @@ define Device/8dev_carambola2
 endef
 TARGET_DEVICES += 8dev_carambola2
 
+define Device/adtran_bsap1880
```

Comment:
Same here: I would go for 18x0 or 1800, otherwise it's misleading.

Reply:
This board had the 1880 product number, even if that does not *seem* consistent with the model numbers it was used in. I'm inclined to NAK this request for now.